grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -155,7 +155,7 @@ class ProductAvailabilityService {
             SELECT
                 pli.binLocation,
                 pli.inventoryItem,
-                sum(pli.quantity)
+                sum(pli.quantity)*(count(distinct pl.requisition) + count(distinct pl.order) / count(*))
```

Comment:
Why you did not use `pli.requisitionItem` and `pli.orderItem`?
Also I think you should add brackets before dividing by the count(*)

Reply:
I checked that with `pli.requisitionItem` and `pli.orderItem` it works too, so I changed it.